testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/UserTest.java

```diff
@@ -1668,4 +1675,35 @@ public void testDeleteCredentials() {
         user.resetPassword(credPasswd);
         Assert.assertEquals(1, user.credentials().size());
     }
+    
+    @Test
+    public void testGetGroupsForUserFullRepresentation() {
+       
+        RealmResource realm = adminClient.realms().realm("test");
+        GroupsResource groups = realm.groups();
+        
+        String userName = "brief-grouptest-user";
+        String groupName = "groupWithAttribute";
+        Map<String, List<String>> attributes = new HashMap<String, List<String>>();
+        attributes.put("attribute1", Arrays.asList("attribute1","attribute2"));
+        
+        try (Response response = groups.add(GroupBuilder.create().name(groupName).attributes(attributes).build())) {
+            String groupId = ApiUtil.getCreatedId(response);
+            
+            UsersResource users = realm.users();
+
+            UserRepresentation userRepresentation = UserBuilder.create()
```

Comment:
It seems this user is never removed. You can use Creator.create(RealmResource, UserRepresentation) that handles cleanup automatically, see e.g. https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/SAMLServletAdapterTest.java#L924. Feel free to use Creator.create(RealmResource, GroupRepresentation) for group similarly.

Reply:
done, for group too